ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -150,6 +150,10 @@ final class StandaloneApiServer(
 
   private def writePortFile(port: Port): Unit =
     config.portFile.foreach { path =>
-      Files.write(path, Seq(port.toString).asJava)
+      if (Files.exists(path)) { sys.error(s"Portfile ${path} already exists.") } else {
```

Comment:
Can we use `Portfiles.write` here and fix that to be atomic instead?

Reply:
sure. Is there any reason why `Portfiles` wasn't used before here?

Reply:
It got created after this piece of code was written and nobody bothered to switch this over (and we didn’t agree on changing the behavior in sandbox at the time).